### PR TITLE
Javascript/javascript2/week1

### PR DESCRIPTION
**Pull request by Elia-Git**
javascript/javascript2/week1 Homework

### CODE REVIEW

File: javascript/javascript2/week1/sprit-animal.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "The joyfull Dragonfly",
+    "The angery Lion",
+    "The peacefull Panda",
+    "The driving force Horse",
+    "The creative Spider",
+    "The humble Turtle",
+];
+const users = [];
+button.addEventListener("click", function(event) {
+    const randomNumber = Math.floor(Math.random() * 12);
```
Comment (florinmaticu) [inline javascript/javascript2/week1/sprit-animal.js:20]: Don't put random constants in your code.
```suggestion
    const randomNumber = Math.floor(Math.random() * animals.length);
```

File: javascript/javascript2/week1/danish-letters-finder.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+function findDanishLetters(str) {
+    let countA = 0;
```
Comment (florinmaticu) [inline javascript/javascript2/week1/danish-letters-finder.js:2]: the variable names are not okay...they mean nothing. Also you could think to use an object that has as keys the danish characters and the values as the number of their occurences. E.g `danishCharacters = {æ:0 , å:0, ø:0  }`
```suggestion
    let countA = 0;
```

File: javascript/javascript2/week1/sprit-animal.js (hunk 1 above)
Comment (Elia-Git) [reply javascript/javascript2/week1/sprit-animal.js]: fixed

File: javascript/javascript2/week1/danish-letters-finder.js (hunk 2 above)
Comment (Elia-Git) [reply javascript/javascript2/week1/danish-letters-finder.js]: This homework is also implemented. Do you have time to review it?

**Pull request merged**